packages/salesforce-adapter/src/metadata_deploy.ts

```diff
@@ -197,13 +200,22 @@ const isUnFoundDelete = (message: DeployMessage, deletionsPackageName: string):
 const processDeployResponse = (
   result: SFDeployResult,
   deletionsPackageName: string,
-  changeDeployedIds: Record<string, MetadataIdsMap>,
-  checkOnly: boolean,
+  typeAndNameToElemId: Record<string, NameToElemIDMap>
```

Comment:
```suggestion
  deployedIdsByType: Record<string, NameToElemIDMap>
```
I find this name quite confusing